### ISSUE TITLE
Publish this version in the vaadin directory

### ISSUE THREAD

**Issue by a-schild** [ylno/ylno-vaadin-dcharts#1]
Finding and installing the addon is not as easy as it could be.
A chance you could publish it in the vaadin directories?

**Comment by ylno** [comment]
Thank you for the hint. The best way would be to get the original author to fix the issues. If i upload this fork there are two plugins which are the same.

**Comment by a-schild** [comment]
Yes, but since the original author did not respond on the vaadin adon page, I think it's abandoned by the user. Perhaps there is a way to "take over" the old entry? (You might ask vaadin, there a a few other addons with the same problem in the directory)

**Comment by pethers** [comment]
Nice to find other people interesting in continue to  @use vaadin-dcharts

Have also found the need to fork my own version, to bad the original author do not respond but also thanks for all his effort.

Only updated jqplot to 1.0.9.d96a669 and built with 7.7.4 , used version 1.7.1-hack23

Maven repository https://www.hack23.com/nexus/repository/maven-public/ 

`
<dependency>
 <groupId>org.dussan.vaadin</groupId>
 <artifactId>dcharts</artifactId>
 <version>1.7.1-hack23</version>
</dependency>
`

Only contains the fixes in https://github.com/pethers/dchartsclone/commit/a6022b8b13cdef8bfff105f8a3074787fc86c6be , but built locally before I created a github fork. 

Would be really nice with vaadin 8 support, know that only 8.0.0.beta1 is released now. But always good to be prepared.
